City-of-Helsinki/helsinki-paatokset
# PP-97: Adjust admin and content producer permission for Päätökset site.

*To test:*
- Checkout branch
- Run `make up && make fresh`
- Login as the root account and create a new user with the "Ylläpitäjä" role
- Login with the newly created account
- Check that you have permissions to create content (especially Päätökset related nodes, terms, media and other entities)
- Editing configuration options should mostly be blocked, since I assume these would be handled locally through the root account
